<launch>
  <param name="states_file" type="string" value="$(find quad_arm_trajectory_generation)/data/states.txt"/>
  <param name="robot_description" textfile="$(find quad_arm_trajectory_generation)/urdf/matrice.urdf"/>
  <param name="controls_file" type="string" value="$(find quad_arm_trajectory_generation)/data/controls.txt"/>

  <rosparam command="load" file="$(find quad_arm_trajectory_generation)/config/place_params.yaml"/>

  <node type="quad" pkg="quad_arm_trajectory_generation" name="quad" output="screen"/>

  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find urdf_tutorial)/urdf.rviz" required="true" />
</launch>
